apollo-runtime/src/main/java/com/apollographql/android/cache/normalized/NormalizedCacheStore.java

```diff
@@ -0,0 +1,17 @@
+package com.apollographql.android.cache.normalized;
+
+import java.util.Collection;
+
+public abstract class NormalizedCacheStore {
```

Comment:
I think that broken cache, should only fail in one case when cache only policy is set.
Other than that if cache is broken for some reason, we should always hit network.
Of course we need to log that as error for user to debug, plus user should understand if response is taken from cache or from network, to proper handle this.